mrp_unbuild_cust_alu_analytics/models/mrp_unbuild_process_type.py

```diff
@@ -7,6 +7,11 @@ class MrpUnbuildProcessType(models.Model):
     _name = "mrp.unbuild.process.type"
     _description = 'Unbuild Process Type'
 
+    company_id = fields.Many2one(
+        comodel_name="res.company",
+        default=lambda self: self.env.company,
+        store=True,
```

Comment:
`store` attribute has no sense (this is a direct field, not a compute one), please remove it.